### ISSUE TITLE
Simple theme chooser for each page

### ISSUE THREAD

**Issue by dawidxt** [pagekit/pagekit#417]
Inside PageKit I am missing option where admin while creating new page, can pick a different theme.pnp layout to each page. If the theme is not picked - it uses the default.php . 

I know it would propably cost You recostruction of main idea of theming inside PageKit, but I think this option will allow frontend developers make more flexible and less boilerplate pages. 

Going into more details:

theme structure:
- assets/
- default.php (theme used by default by the CMS when none other is chosen)
- homepage.php ( optional homepage theme, made by developer to customize the look of homepage)
- contact.php (with for example two columns body, where left body could be a map or form, and right column could be a dynamic text)

Maybe I am wrong, and people with better knowledge can override this proces going hard way? anyway
I was wondering why there is no such option inside PageKit, while other smaller CMS's have it.

Sorry for my English, If I wasn't clear enough, I am providing list of small cms'es that has such option:
- GetGrav
- KirbyCMS
- Monstra CMS
- Get Simple
- Anchor CMS
- Wolf CMS

**Comment by malles** [comment]
This can be implemented from the theme itself. The theme builder can choose to offer themes or layouts per page.

**Comment by dawidxt** [comment]
Yeah, I was thinking that there might be such option deep inside the hood, but is that more comfortable and flexible as having such option inside dashboard? I don't think so.

**Comment by ste93cry** [comment]
:+1: something like WordPress page templates feature would be cool

**Comment by BrightdayStudio** [comment]
👍
